modules/test-runner/src/deposit/deposit.test.ts

```diff
@@ -219,9 +239,20 @@ describe("Deposits", () => {
       assetId: tokenAddress,
       node: Zero,
     };
-    await client.deposit({ amount: TWO, assetId: ethExpected.assetId });
+    let response = await client.deposit({ amount: TWO, assetId: ethExpected.assetId });
+    if (response.completed) {
```

Comment:
In what cases would we *not* have a `response.completed` here?

Reply:
Backwards compatibility. In old clients, the deposit method would wait for the full deposit to complete.